docs/src/customprocessing.md

```diff
@@ -45,3 +46,58 @@ now simply give this function to the generator, for example:
 ```julia
 Literate.markdown("input.jl", "outputdir"; preprocess = update_date)
 ```
+
+## Example: Replacing `include` calls with included code
+Let's say that we have some individual example files `file1, file2, ...` etc.
+that are _runnable_ and also following the style of Literate. These files could be for example used in the test suite of your package.
+
+We want to group them all into a single page in our documentation, but we
+do not want to copy paste the content of `file1, ...` for robustness: the files are included in the test suite and some changes may occur to them. We want these changes to also be reflected in the documentation.
+
+A very easy way to do this is using `preprocess` to interchange `include` statements with file content. First, create a runnable `.jl` following the format of Literate (the following example comes from the documentation of the Julia package [`TimeseriesPrediction`](https://github.com/JuliaDynamics/TimeseriesPrediction.jl), which uses this approach to create some pages)
+```julia
+# # Spatio-Temporal Prediction Examples
```

Comment:
While I appreciate this real-world example, perhaps it is better with something simpler, for example
```
# # Replace includes
# This is an example to replace `include` calls with the actual file content.

include("file1.jl")
include("file2.jl")
```
I think that the message might be clearer that way? It is still good to link to your package of course, as an example.